components/ProfilePageComponent/SkillsSection.tsx

```diff
@@ -47,14 +47,18 @@ const SkillsSection = () => {
         setIsOpen(false);
     }
 
-    useEffect(() => {
+    const getData = useCallback(() => {
         GetData('/api/skills?userId=' + _id).then((res: any) => {
             console.log(res);
             const { data = [], status } = res;
             if (status == 201) setSkillTags(data);
         });
     }, []);
```

Comment:
@JRakin _id will be missing in first render. so add _id as dependency

Reply:
@sifatul sorry my bad didn't notice fixed it